### ISSUE TITLE
Features needed for CMSSW_10_3_1 for initial HI data taking

### ISSUE THREAD

**Issue by fabiocos** [cms-sw/cmssw#25040]
I would like to collect in this issue the list of URGENT features that are needed in order to deploy the forthcoming CMSSW_10_3_1 in production for data taking. The target for such a release is after the ORP of 2018/10/30

Status on 2018/11/1 10:00

as discussed at ORP of 10/30 https://docs.google.com/spreadsheets/d/1TJWAJEXhXcnhw2WTyECGukf6bD7kixV1yB7Cets5mHc/edit#gid=733750101 the following PRs have been integrated:

- [x] #24928 skims for 2018 HI
- [x] #24976 fix for AlcaReco HI
- [x] #24987 update to uGT unpacker for HI
- [x] #25010 update to CaloL2 O2O for L1T HI

Further required updates discussed:

- [x] #25065 update to HLT for new L1T objects (10_4_X #25064)
- [ ] #25066 switch RAW repack to LZMA compression (10_4_X #25067) **ON HOLD FOR NOW**
- [x] #24888 fixes for online beamspot HI 2018 (10_4_X #25073, rebase of #24887) 

Updates linked to HI HLT menu in the release are not crucial for 10_3_1 to operate at T0/P5

**Comment by slava77** [comment]
@icali @mandrenguyen 
in case you haven't seen this already

**Comment by fabiocos** [comment]
@slava77 thank you, I have updated the list accordingly (se above)

**Comment by fabiocos** [comment]
@slava77 @tommasoboccali @davidlange6 based on the discussion https://hypernews.cern.ch/HyperNews/CMS/get/tier0-Ops/2055/1/3/1/1/1/1/1/1/1/1/1/2/1/1/1/1.html
I will keep for the time being #25066 / #25067 in stand-by, and will not merge them for 10_3_1

**Comment by fabiocos** [comment]
@boudoul Based on the current situation of the integration, I will move forward with the build in parallel of the latest IB. Possible DQM additions will come in a second moment.

**Comment by boudoul** [comment]
> @boudoul Based on the current situation of the integration, I will move forward with the build in parallel of the latest IB. Possible DQM additions will come in a second moment.

Absolutely, thanks @fabiocos !
